packages/components/src/Tag/Tag.component.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Tag as CoralTag } from '@talend/design-system';
+
+/**
+ * Proxy to https://design.talend.com/?path=/docs/components-tag--default-story#tag
+ */
+const Tag = ({ bsStyle, ...rest }) => {
+	let StyledTag = CoralTag;
+	switch (bsStyle?.toLowerCase()) {
+		case 'info':
+			StyledTag = CoralTag.Information;
+			break;
+		case 'success':
+			StyledTag = CoralTag.Success;
+			break;
+		case 'warning':
+			StyledTag = CoralTag.Warning;
+			break;
+		case 'danger':
+			StyledTag = CoralTag.Destructive;
+			break;
+		default:
+			break;
+	}
+	return <StyledTag {...rest} />;
+};
```

Comment:
Why this `new` component ? Shouldn't it be a simple proxy to DS instead of offering a new api ? I mean it's not a component that exists today in tui, so we are not stucked with an existing api

Reply:
Ok let's get rid of it!

Reply:
I can't, actually! If I don't have this component as a proxy, I need to duplicate the mapping bootstrap v3 -> Design System several times!